test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -15,3 +15,76 @@
 
 // CHECK: struct HasDeletedOperator {
 // CHECK: }
+
+
+// CHECK: struct ReadWriteIntArray {
+// CHECK:   struct NestedIntArray {
+// CHECK:     @available(*, unavailable, message: "use subscript")
+// CHECK:     mutating func __operatorSubscriptConst(_ x: Int32) -> UnsafePointer<Int32>
```

Comment:
WDYT about making these helper functions transparent so they're always inlined?

Reply:
Great idea, thanks! I've made the generated subscript accessors transparent. Would transparency have any effect on the functions imported from C++ directly, like `__operatorSubscriptConst`? (and wouldn't it break if the underlying C++ implementation changes)

Reply:
Sorry, no, you're right. The only things that should be transparent are the accessors. I think it would "work" to mark `__operatorSubscriptConst` as transparent, but it wouldn't get inlined. That happens at IRGen time (and I don't think the attribute is still around during IRGen).